src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferAgent.java

```diff
@@ -79,7 +79,8 @@
   // with 4 threads by default
   private Set<String> smallSourceFiles;
 
-  private static final int BIG_FILE_THRESHOLD = 64 * 1024 * 1024;
+  /* Default value, in bytes */
+  private int multipartUploadThreshold = 200 * 1024 * 1024;
```

Comment:
So I guess you write code here in case GS doesn't return THRESHOLD parameter in PUT? In either case, the value "200" should be in a constant variable so that it can be shared.

Reply:
I added back in the constant variable BIG_FILE_THRESHOLD_MB and then multipartUploadThreshold variable now uses that value.